internal/cmd/users.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+func ListServerUsersTable(users []*core.ServerUserInfo) {
```

Comment:
I would consider returning a string instead of printing.
(To build the string you should probably use a strings.Builder).